crates/sui-indexer-alt-graphql/src/extensions/logging.rs

```diff
@@ -64,22 +70,25 @@ impl Extension for LoggingExt {
         self.metrics.queries_in_flight.dec();
 
         // SAFETY: This is set by `prepare_request`.
-        let Session(uuid, addr) = self.session.lock().unwrap().unwrap();
+        let Session {
+            request_id,
+            client_addr,
+        } = self.session.lock().unwrap().unwrap();
 
         if response.is_ok() {
-            info!(%uuid, %addr, elapsed_ms, "Request succeeded");
+            info!(%request_id, %client_addr, elapsed_ms, "Request succeeded");
```

Comment:
Could you keep the names as they were? These changes have an effect on the tracing output, because the variable names are also used as the label names in logs.